исправление проблем в TimetableOfClasses\AddAcademicTitle.cs

SlavaVinogradov opened the pull request:
Students-of-the-city-of-Kostroma#1472  Fix 11 Style, 1 Complexity issues in TimetableOfClasses\AddAcademicTitle.cs

SlavaVinogradov committed:
Update AddAcademicTitle.cs

SlavaVinogradov committed:
Update AddAcademicTitle.cs

SlavaVinogradov committed:
Update AddAcademicTitle.cs

SlavaVinogradov committed:
Update AddAcademicTitle.cs

YuriSilenok commented on TimetableOfClasses/AddAcademicTitle.cs:
```diff
@@ -78,57 +55,26 @@ private void btCreateAndClose_Click(object sender, EventArgs e)
+            catch (Exception)
+            {
+                MessageBox.Show("Некорректно заполнены поля", "Ошибка");
             }
```
оба обработчика исключений можно заменить на один с выводом сообщения исключения

YuriSilenok replied:
проблема осталась

SlavaVinogradov committed:
Update AddAcademicTitle.cs

SlavaVinogradov reviewed (commented):
Students-of-the-city-of-Kostroma#1472  Fix 11 Style, 1 Complexity issues in TimetableOfClasses\AddAcademicTitle.cs

YuriSilenok commented:
> #1472 Fix 11 Style, 1 Complexity issues in TimetableOfClasses\AddAcademicTitle.cs

это не комментарий к коммиту

SlavaVinogradov committed:
Исправления

Students-of-the-city-of-Kostroma#1472  Fix 11 Style, 1 Complexity issues in TimetableOfClasses\AddAcademicTitle.cs

YuriSilenok commented:
теперь получилось

SlavaVinogradov commented:
Но я добавил комментарий к новому коммиту.

YuriSilenok commented:
этого достаточно

YuriSilenok reviewed (changes requested):
я ожидаю что вы будете выводить сообщение возвращаемое исключением

SlavaVinogradov committed:
Update AddAcademicTitle.cs

Students-of-the-city-of-Kostroma#1472  Fix 11 Style, 1 Complexity issues in TimetableOfClasses\AddAcademicTitle.cs
